src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/ConfigurationEndpoints.cs

```diff
@@ -32,3 +43,5 @@ public record OidcAuthenticationSettings
   public string Scope { get; set; } = null!;
   public string? Idp { get; set; }
 }
+
+public record Province(string ProvinceId, string ProvinceName);
```

Comment:
Do you want the front end to know about the ID or is it just the standard abbreviation?

Reply:
It is the id from dynamics like all other lookup values